src/client/game.cpp

```diff
@@ -3264,7 +3270,7 @@ PointedThing Game::updatePointedThing(
 
 void Game::handlePointingAtNothing(const ItemStack &playerItem)
 {
-	infostream << "Right Clicked in Air" << std::endl;
+	infostream << "Attempted to place item while pointing at nothing" << std::endl;
```

Comment:
This is incorrect, this function handles both LMB and RMB while pointing at nodes.